cli/src/Api.ts

```diff
@@ -0,0 +1,135 @@
+import BN from 'bn.js';
+import { registerJoystreamTypes } from '@joystream/types';
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import { formatBalance } from '@polkadot/util';
+import { Balance, Hash } from '@polkadot/types/interfaces';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { AccountBalances, CouncilInfoObj, CouncilInfoTuple, AccountSummary, createCouncilInfoObj } from './Types';
+import { DerivedFees, DerivedBalances } from '@polkadot/api-derive/types';
+import { CLIError } from '@oclif/errors';
+import ExitCodes from './ExitCodes';
+
+const API_URL = process.env.WS_URL || (
```

Comment:
I think we are getting to the stage where the tool should allow the user to provide their own URL.

It does not need to be addressed in this PR, you can open a new one for this.